abci/client/grpc_client.go

```diff
@@ -89,7 +91,14 @@ func (cli *grpcClient) OnStart() error {
 
 RETRY_LOOP:
 	for {
-		conn, err := grpc.Dial(cli.addr, grpc.WithTransportCredentials(insecure.NewCredentials()), grpc.WithContextDialer(dialerFunc)) //nolint:staticcheck
```

Comment:
Previously this was created without any `MaxCallRecvMsgSize` so it defaulted to 4MB